dbg.h

```diff
@@ -550,7 +550,12 @@ inline bool pretty_print(std::ostream& stream, const time&) {
   const auto us =
       duration_cast<microseconds>(now.time_since_epoch()).count() % 1000000;
   const auto hms = system_clock::to_time_t(now);
+#if _MSC_VER >= 1600
```

Comment:
Why do we want to do this for `_MSC_VER >= 1600` only and not everywhere?

Reply:
localtime_s was introduced in vs2010(_MSC_VER >= 1600), so just replace localtime with localtime_s here.

Reply:
Ok, and how about non-MSV environments?

Reply:
non-MSV environments also used old #else statements. 
added macro just fix MSV environments error. /Sa my first comment.